client/allocrunner/networking_cni.go

```diff
@@ -347,8 +577,9 @@ func (c *cniNetworkConfigurator) ensureCNIInitialized() error {
 
 // getPortMapping builds a list of portMapping structs that are used as the
 // portmapping capability arguments for the portmap CNI plugin
-func getPortMapping(alloc *structs.Allocation, ignoreHostIP bool) []cni.PortMapping {
+func getPortMapping(alloc *structs.Allocation, ignoreHostIP bool) ([]cni.PortMapping, map[string]int) {
 	var ports []cni.PortMapping
+	labels := map[string]int{}
 
```

Comment:
the comment and the `labels` variable could use some clarification around what information the var is holding (label -> index)

just thinking: one could even imagine creating a small custom data structure to combine both the port mapping slice and label map so that consumers don't need to think about the intricate relationship between the two values returned from this function

Reply:
Done in 4fef82e8e231f0d0509b88e91a9b40903aabf9cc